clients/linux/src/backend.rs

```diff
@@ -342,16 +361,69 @@ impl LbCore {
         Ok((meta, String::from_utf8_lossy(&decrypted).to_string()))
     }
 
-    pub fn sync_status(&self) -> LbResult<String> {
+    pub fn sync_status(&self, just_synced: bool) -> LbResult<String> {
         match self.get_last_synced()? {
             0 => Ok("✘  Never synced.".to_string()),
             _ => {
-                let work = self.calculate_work()?;
+                let work = match calculate_work(&self.config) {
+                    Ok(ok) => ok,
+                    Err(err) => {
+                        if let UiError(CalculateWorkError::CouldNotReachServer) = err {
+                            return Ok("You are offline".to_string());
+                        }
+
+                        return Err(match_core_err!(err, CalculateWorkError,
+                            CouldNotReachServer => uerr!("Unable to connect to the server."),
+                            ClientUpdateRequired => uerr!("Client upgrade required."),
+                            NoAccount => uerr!("No account found."),
+                            @Unexpected(msg) => return Err(progerr!("{}", msg)),
+                        ));
+                    }
+                };
+
                 let n_files = work.work_units.len();
+
+                if just_synced {
+                    return Ok(match n_files {
+                        0 => "✔  Synced.".to_string(),
+                        _ => format!(
+                            "<b>{}</b>  file{} not synced.",
+                            n_files,
+                            if n_files == 1 { "" } else { "s" }
+                        ),
+                    });
+                }
+
+                let usage = self.local_and_server_usage()?;
+
+                let data_cap = match usage.data_cap.find(' ') {
+                    None => Err(progerr!("Could not parse data cap.")),
+                    Some(index) => usage.data_cap[..index]
+                        .parse::<f32>()
+                        .map_err(|err| progerr!("Could not parse data cap: {:?}", err)),
+                }?;
+
+                let total_usage = match usage.server_usage.find(' ') {
+                    None => Err(progerr!("Could not parse total usage.")),
+                    Some(index) => usage.server_usage[..index]
+                        .parse::<f32>()
+                        .map_err(|err| progerr!("Could not parse total usage: {:?}", err)),
+                }?;
+
+                if total_usage / data_cap > USAGE_WARNING_THRESHOLD {
+                    return Ok(format!(
+                        "Warning, you are currently over {}% usage.",
```

Comment:
I think this should say:
`n` _b of `m` _b remaining!

When they mouse over it should say:
You are running out of space, go to settings to buy more!

@tvanderstad something to note for UX docs, there's already some stuff spec'd out in billing.md. But it's probably a good time to start thinking about these flows.